test/evergreen.yml

```diff
@@ -45,6 +45,12 @@ functions:
     params:
       script: |
         git clone https://github.com/wiredtiger/mongo-tests
+  "fetch mongodb bin" :
+    command: shell.exec
```

Comment:
Our common shell settings include 'set -o errexit & set -o verbose'. Should they be added for this shell command?

Reply:
Yes, thanks for spotting it.
I used the `fetch mongo-tests repo` as a template. I will update that ask as well.

Reply:
4b098c7fcf1aac1f460290aa4f16a298cc5f9000